src/ethereum_clis/clis/nimbus.py

```diff
@@ -66,69 +67,62 @@ class NimbusExceptionMapper(ExceptionMapper):
     """Translate between EEST exceptions and error strings returned by Nimbus."""
 
     mapping_substring: ClassVar[Dict[ExceptionBase, str]] = {
-        TransactionException.TYPE_4_TX_CONTRACT_CREATION: (
-            "set code transaction must not be a create transaction"
+        # TODO: these need to be fixed / incorrectly mapped
+        TransactionException.INSUFFICIENT_MAX_FEE_PER_GAS: (
```

Comment:
if the exception string is different to the exception meaning. we can create another exception code for nimbus, and then in the test add this to the list of possible exceptions  

`TransactionException.INSUFFICIENT_MAX_FEE_PER_GAS| TransactionException.ZERO_GAS_FEE`

but only if the exception logic by nimbus is correct